components/match2/commons/player_ext.lua

```diff
@@ -239,6 +239,12 @@ function PlayerExt.syncTeam(pageName, template, options)
 		template = template ~= 'noteam' and template or nil,
 	}
 
+	-- catch some edge cases for pageVarEntry
```

Comment:
Can it be updated with somewhat more info than just "some edge cases" (ie what edge cases). Same goes for PR Title

Reply:
updated